docs/process.md

```diff
@@ -1691,6 +1691,8 @@ To illustrate the differences between error strategies, it can be helpful to ima
 - `finish`: Nextflow will allow any existing jobs to conclude but not submit any more and report an error
 - `ignore`: Nextflow will proceed to continue submitting processes for the remaining 95 samples and proceed to completion. Nextflow will ignore the error and report a successful pipeline completion
 - `ignore` and `workflow.failOnIgnore`: Nextflow will proceed to continue submitting processes for the remaining 95 samples and proceed to completion, then exit with an error status and report the error
+
```

Comment:
Suggestions for above lines (since I was too slow in the last PR, and I don't have permission to push to this one):

```
To illustrate the differences between error strategies, consider the following example. Let's say you are analyzing 96 patient samples and the data from one is corrupted, causing the associated task to fail. The different ways to handle this failure are as follows:

- **errorStrategy `terminate`**: Nextflow will cancel any other ongoing tasks at the time of the failure, exit the pipeline, and report an error.
- **errorStrategy `finish`**: Nextflow will allow any other existing tasks to conclude (but not submit any more) and report an error.
- **errorStrategy `ignore`**: Nextflow will continue submitting tasks for the remaining 95 samples, complete the workflow, and report a successful pipeline completion.
- **errorStrategy `ignore` and `workflow.failOnIgnore` set to `true` in configuration**: The same behavior as setting the errorStrategy alone, except the pipeline will return an exit status of -1 and report an error.
```

Reply:
Works for me if @adamrtalbot is happy?

Reply:
I like it 👍🏻 

"same as setting the `ignore` errorStrategy"

on the last line maybe. But otherwise great 👍🏻

Reply:
resolved in https://github.com/nextflow-io/nextflow/pull/5123